src/PhpSpreadsheet/Writer/Xls/Xf.php

```diff
@@ -535,14 +516,10 @@ private static function mapLocked($locked)
     private static function mapHidden($hidden)
     {
         switch ($hidden) {
-            case Protection::PROTECTION_INHERIT:
-                return 0;
             case Protection::PROTECTION_PROTECTED:
                 return 1;
-            case Protection::PROTECTION_UNPROTECTED:
-                return 0;
-            default:
-                return 0;
         }
+
+        return 0;
```

Comment:
As comment above for `mapLocked()`

Reply:
Likewise - see resolution above for mapLocked.